Dockerfile

```diff
@@ -1,11 +1,18 @@
 FROM debian:stable-slim
+SHELL [ "/bin/sh", "-ec" ]
 
-RUN apt-get update && apt-get -uy upgrade
-RUN apt-get -y install ca-certificates && update-ca-certificates
```

Comment:
Is running `update-ca-certificates` no longer necessary?

Reply:
This is done automatically by postinst script in "ca-certificates" package.